src/Ploi/Resources/Server.php

```diff
@@ -103,9 +103,61 @@ public function create(
         return $response->getData();
     }
 
-    public function createCustom(): stdClass
+    public function createCustom(string $ip, array $options): stdClass
     {
         $endpoint = $this->getEndpoint() . '/custom';
+
+        $defaults = [
+            'ip' => $ip,
+            'type' => 'server',
+            'database_type' => 'mysql',
+            'php_version' => '7.4',
+        ];
+
+        $options = [
+            'body' => json_encode(array_merge($defaults, $options)),
+        ];
+
+        $response = $this->getPloi()->makeAPICall($endpoint, 'post', $options);
+
+        $data = $response->getJson();
+        $this->setId($data->id);
+
+        return $data;
+    }
+
+    public function startInstallation(string $url = null)
+    {
+        $id = $this->getId();
+
+        if (!$url && !$id) {
+            throw new RequiresId("This endpoint requires an ID. Supply an ID or a valid installation url.");
+        }
+
+        $endpoint = $url ?: $this->getEndpoint() . "/custom/{$id}/start";
+
+        $response = $this->getPloi()->makeAPICall($endpoint, 'post');
+
+        $data = $response->getJson();
+
+        return $data;
+    }
+
+    public function sshKeys(int $id = null): array
```

Comment:
https://developers.ploi.io/ssh-keys/list-ssh-keys

Reply:
As there are more methods for SSH Keys it might be best to do this as a separate resource like we do with sites and backups.
So then it would be like:
$ploi->servers(123)->sshKeys()->get();

Reply:
Good point, I've removed that for now